blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/outputs/handlers/HttpRollupsQueryHandlerIntegrationTest.java

```diff
@@ -84,7 +84,7 @@ public void testSingleplotQueryWithEnum() throws Exception {
         final String metric_name = "enum_metric_test" + postfix;
 
         // post multi metrics for ingestion and verify
-        HttpResponse response = postMetric(tenant_id, postAggregatedPath, "sample_enums_payload.json", now, postfix );
+        HttpResponse response = postMetric(tenant_id, postAggregatedPath, "sample_single_enum_payload.json", now, postfix );
```

Comment:
This test was posting several enum values to 1 timestamp (values of v1, v2, v3).  Only 1 of them is recorded on our local boxes as well as on jdk7 on travis, the v3 value.  However, on travis only v2 is returned under jdk 8.  I changed this so we can verify the default behavior on travis jdk8.  I didn't mess with this issue as we will eventually be getting rid of enums.  I can document this in the code.

Now, the multiple eums values are recorded in elasticsearch as the enum values are turned into metric names, so they can be queried. This is tested in HttpEnumIntegrationTest.testMetricIngestionWithEnum().  So yeah, enums are whack.  Only one value is saved for a timestamp, but all submitted values, regardless if they are the same timestamp are recorded in elasticsearch.